### ISSUE TITLE
Add color bar component for MapPlot3D plot

### ISSUE THREAD

**Issue by svniemeijer** [stcorp/atmosphere-virtual-lab#8]
This should be similar to https://github.com/stcorp/leaflet-gl-vector-layer/issues/4 but then ideally using an existing VTK component for showing color bars.

**Comment by srepmub** [comment]
there is already a VTK color-bar per layer (which perhaps we should show instead of hide by default?, so I guess this is more about interactive selection of layers and such, so on the js side..?

**Comment by svniemeijer** [comment]
No. This is first and foremost about being able to show/hide the color bar programmatically. Having interactive control over this might be something for later.

**Comment by svniemeijer** [comment]
I see. We indeed have a color bar like that.
We might want to think about providing our own color bar if this doesn't work nicely (maybe even reuse what was done in VISAN).
